inert.js

```diff
@@ -633,6 +633,8 @@ function addInertStyle(node) {
                       "  -webkit-user-select: none;\n" +
                       "  -moz-user-select: none;\n" +
                       "  -ms-user-select: none;\n" +
+                      "  overflow: hidden !important;\n" +
```

Comment:
It does cause a repaint, but only once per inert root, rather than once per element inside the inert root.

The Windows issue is trickier, though. It seems like there's no good solution here currently.